src/course/syllabus/apprenticeship/full-stack-app/schedule.md

```diff
@@ -2,6 +2,11 @@
 layout: schedule
 schedule:
   monday:
+    - name: Learn Next.js
+      url: https://nextjs.org/learn/basics/create-nextjs-app
+      type: workshop
+      start: 9:45
+      end: 18:00
   tuesday:
```

Comment:
I think the empty challenge slot should be additional project time - am aware last cohort seemed to have the whole day for the project (this time there is TfB scheduled for part of the afternoon).

Given the above, maybe we should also reduce down the discussion time in the morning - although I can do this add hoc on the day rather than changing the schedule